propagation/trace_context.go

```diff
@@ -143,6 +132,12 @@ func (tc TraceContext) extract(carrier TextMapCarrier) trace.SpanContext {
 	// Clear all flags other than the trace-context supported sampling bit.
 	sc.TraceFlags = opts[0] & trace.FlagsSampled
 
+	ts, err := parseTraceState(carrier.Get(tracestateHeader))
+	if err != nil {
+		return trace.SpanContext{}
```

Comment:
Quote from [Trace Context](https://www.w3.org/TR/trace-context/#tracestate-header)

> failure to parse tracestate MUST NOT affect the parsing of traceparent.

I believe we should return `traceparent` that is already parsed.

Reply:
Good call, updated.